index.bs

```diff
@@ -181,7 +181,7 @@ the Generic Sensor API [[!GENERIC-SENSOR]].
 Permissions Policy integration {#permissions-policy-integration}
 ==============================
 
-This specification defines a [=policy-controlled feature=] identified by the string "<code><dfn data-lt="magnetometer-feature" export>magnetometer</dfn></code>". Its [=default allowlist=] is "`self`".
+This specification utilizes the [=policy-controlled feature=] identified by the string "<code><a data-lt="magnetometer-feature">magnetometer</a></code>".
```

Comment:
The definition of the "magnetometer" permission below needs to be updated to a reference as well.